TabloidMVC/Repositories/UserProfileRepository.cs

```diff
@@ -8,6 +11,46 @@ public class UserProfileRepository : BaseRepository, IUserProfileRepository
     {
         public UserProfileRepository(IConfiguration config) : base(config) { }
 
+
```

Comment:
This GetAllUsers method will conflict with my method. My "GetAll" method is the same, except additionally adds a UserType object to the model from the database. Other than that, it should be the same and not break the function of your code when we merge.